lib/jar.ts

```diff
@@ -0,0 +1,22 @@
+import * as path from 'path';
+import * as crypto from 'crypto';
+import * as fs from 'fs';
+
+export function getTempPomPathForJar(root: string, jar: string): string {
+  const sha1 = getSha1ForJar(path.join(root, jar));
+
+  // return path to root of pom.xml
+  return '';
+}
+
+export function isJar(file: string) {
+  return file.match(/\.(([jw]ar)|(zip))$/);
+}
+
+export function getSha1ForJar(filePath) {
+  const fileContents = fs.readFileSync(filePath);
+  return crypto
+    .createHash('sha1')
```

Comment:
looking at example here: https://nodejs.org/api/crypto.html#crypto_crypto_createhash_algorithm_options
this has could be a const, instead of creating on each function call